guides/common/attributes-foreman-el.adoc

```diff
@@ -1,4 +1,6 @@
 // Overrides for foreman-el build
+:dnf-module: foreman:el8
+:dnf-modules: {dnf-module}
```

Comment:
We have to be careful with the EL version on 3.1-3.3, because we can have the Project on both EL7 & EL8 and I'm assuming that the dnf module as listed above is applicable only to EL8. Therefore, we need to add one more instruction for EL7. Pls, check with a SME.